01_Data/tsconfig.json

```diff
@@ -1,20 +1,17 @@
 {
-  "compilerOptions": {
-    "target": "es6",
-    "module": "commonjs",
-    "experimentalDecorators": true,
-    "emitDecoratorMetadata": true,
-    "declaration": true,
-    "outDir": "lib",
-    "strict": true,
-    "resolveJsonModule": true,
-    "esModuleInterop": true
-  },
+  "extends": "../tsconfig.build.json",
   "include": [
-    "src"
+    "src/**/*.ts",
+    "src/**/*.json"
   ],
-  "exclude": [
-    "node_modules",
-    "**/__tests__/*"
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "composite": true,
+    "rootDir": "./src"
+  },
+  "references": [
+    {
+      "path": "../00_Base"
+    }
```

Comment:
I took a look through what they have for `declarationMap` and it is not 100% clear to me if this will be helpful. In my case, I am able to jump to definitions between modules so long as I run the `npm i` first successfully